modules/registry-configuring-storage-gcp-user-infra.adoc

```diff
@@ -6,14 +6,14 @@
 = Registry storage for GCP with user-provisioned infrastructure
 
 You must set up the storage medium manually and configure the settings in the
-registry CRD.
+registry Custom Resource Definition (CRD).
```

Comment:
Outside of writing an Operator, a user will never do anything with a CustomResourceDefinition object, which is what this is a reference to.

It might be the registry CR? This might refer to a global config object. They're all named `cluster`. Not sure.